includes/core-changes/aspnetcore/3.0/obsolete-apis-removed.md

```diff
@@ -45,8 +45,8 @@ The following types and members were marked as obsolete for ASP.NET Core 2.1 and
 - <xref:Microsoft.AspNetCore.Mvc.Formatters.FormatFilter.%23ctor(Microsoft.Extensions.Options.IOptions{Microsoft.AspNetCore.Mvc.MvcOptions})?displayProperty=fullName>
 - <xref:Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ArrayModelBinder%601.%23ctor(Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder)?displayProperty=fullName>
 - <xref:Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ByteArrayModelBinder.%23ctor?displayProperty=fullName>
-- <xref:Microsoft.AspNetCore.Mvc.ModelBinding.Binders.CollectionModelBinder%601.%23ctor(Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder)?displayProperty=fullName>
-- <xref:Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexTypeModelBinder.%23ctor(System.Collections.Generic.IDictionary{Microsoft.AspNetCore.Mvc.ModelBinding.ModelMetadata,Microsoft.AspNetCore.Mvc.ModelBinding.IModelBinder})?displayProperty=fullName>
+- [Microsoft.AspNetCore.Mvc.ModelBinding.Binders.CollectionModelBinder<TElement>(IModelBinder)](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.modelbinding.binders.collectionmodelbinder-1.-ctor?view=aspnetcore-2.2#Microsoft_AspNetCore_Mvc_ModelBinding_Binders_CollectionModelBinder_1__ctor_Microsoft_AspNetCore_Mvc_ModelBinding_IModelBinder_)
```

Comment:
@scottaddie I only "fixed" 5 links in this file, but I think they are all technically broken since the APIs have been removed. I've enquired if there's a way to xref link to a specific version. Otherwise, I'm not sure if you think it's worth it to go through and add the explicit version (ASP.NET Core 2.2) like I did here, instead of using an xref link.

Reply:
I'm okay with using the explicit version link. That might be the only way to link to a specific version.

Reply:
Okay, could you add this to your work items? I'll also add a feature request to be able to target a specific version in an xref link. (It would also be useful to be able to target any version >= a specific version for new APIs, but that doesn't apply to this PR.)

Reply:
@scottaddie In our sprint meeting this morning I brought this up, and the question was raised, do we even need a link if the API has been removed. How do you feel about simply unlinking these API references?